test/Soil/soil_parameterizations.jl

```diff
@@ -48,22 +49,21 @@ include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
         ν_ss_om = ν_ss_om,
         ν_ss_quartz = ν_ss_quartz,
         ν_ss_gravel = ν_ss_gravel,
-        vg_α = vg_α,
-        vg_n = vg_n,
+        hydrology_cm = hcm,
         K_sat = K_sat,
         S_s = S_s,
         θ_r = θ_r,
         earth_param_set = param_set,
     )
     # Test that the preset params are set properly
-    (; α, β, Ω, γ, vg_m, γT_ref) = parameters
+    (; α, β, Ω, hydrology_cm, γ, γT_ref) = parameters
     @test α == 0.24
     @test β == 18.3
     @test γ == 2.64e-2
     @test Ω == 7.0
     @test γT_ref == 288.0
-    @test vg_m == 1.0 - 1.0 / vg_n
-    @test typeof.([α, β, γ, Ω, vg_m, γT_ref]) == [FT, FT, FT, FT, FT, FT]
+    @test hydrology_cm.m == 1.0 - 1.0 / vg_n
```

Comment:
Should we also test the other fields of `hydrology_cm`?

Reply:
I added a test for S_c, but I think only testing m and S_c (and just S_c) for Brooks Corey is OK. I added these in particular because they are derived parameters in the constructor, not ones that are passed in.

Reply:
I think I may have accidentally unresolved this? I was scrolling on my phone, and it happened so fast.